meta-mender-variscite/recipes-bsp/u-boot/u-boot-variscite-mender-common.inc

```diff
@@ -45,3 +68,9 @@ MENDER_VARISCITE_UBOOT_PRE_SETUP_COMMANDS = " \
 
 MENDER_UBOOT_PRE_SETUP_COMMANDS_var-som-mx6 = "${MENDER_VARISCITE_UBOOT_PRE_SETUP_COMMANDS}"
 MENDER_UBOOT_PRE_SETUP_COMMANDS_imx6ul-var-dart = "${MENDER_VARISCITE_UBOOT_PRE_SETUP_COMMANDS}"
+
+# For imx8mm-var-dart
+MENDER_UBOOT_CONFIG_SYS_MMC_ENV_PART = "0"
```

Comment:
If this is unconditional, e.g. without a machine-type override, then it breaks all other platforms. Please add the machine specifiers.

Reply:
Added in commit 89d22d4a4d6125d4b55b05624735c9b73db84e78